include/crud-posts.php

```diff
@@ -4,12 +4,32 @@
  */
 
 /**
- * Adds actions and filters related to languages when creating, updating or deleting posts
- * Actions a filters used when reaing posts are handled separately
+ * Adds actions and filters related to languages when creating, updating or deleting posts.
+ * Actions an filters used when reading posts are handled separately.
```

Comment:
- Probably 'Actions **and** filters'. 
- 'Used' is very generic. 'Triggered' maybe?
- I suppose that is the reading that **is** handled separately.
- Separately from what?

Reply:
`Separately` from creating, updating or deleting posts events quoted above the comment. It seems clear to me.

Reply:
Ah, you mean that this class does not handle reading posts, whereas its name 'CRUD' might make us think it does...

Okay, I get it now.